src/features/settings/pages/SecurityPage.tsx

```
@@ -97,490 +91,320 @@ export interface SecurityPageProps {
-              onClick={() => void handleResetPassword()}
-              disabled={isResettingPassword}
-              className="text-accent dark:text-accent hover:text-accent-deep dark:hover:text-accent-deep"
-            >
-              {isResettingPassword ? t('common:status.sending', { defaultValue: 'Sending…' }) : t('settings:security.password.resetButton')}
-            </Button>
-          )}
+    <div className={className}>
+      <SettingSection first title="Email address" description="Your login email. Changing this sends a verification link to the new address.">
+        <div className={securityCardClassName}>
+          <div className="form-field">
+            <label className="label">Email</label>
```

ESLint

[error] A form label must be associated with a control.

(jsx-a11y/label-has-associated-control)